test/unit/job_test.rb

```diff
@@ -643,7 +655,7 @@
 
         describe 'with non sidekiq job class' do
           before do
-            @job = Sidekiq::Cron::Job.new(@args.merge(klass: 'CronTestClass'))
+            @job = Sidekiq::Cron::Job.new(@args.merge(klass: 'ActiveJobCronTestClass'))
```

Comment:
This is not clearly visible on diff, but this is in `active_job == true` block thus I'd say it's safe to assume any klass at least mimics ActiveJob interface. This may be a breaking change if someone's using  'active_job == true` in config with non-ActiveJob class because `queue_name` on class might not be available :cry: